tool/tsh/proxy.go

```diff
@@ -316,6 +318,33 @@ func proxySession(ctx context.Context, sess *tracessh.Session) error {
 	return trace.NewAggregate(errs...)
 }
 
+// formatCommand formats command making it suitable for the end user to copy the command and paste it into terminal.
+func formatCommand(cmd *exec.Cmd) string {
```

Comment:
Sorry missed this part in the original PR. I don't think it's a blocker but just curious how this will work out for Connect.

Reply:
Huh, I don't know either, I'd but properly escaping arguments should improve things, not make them worse. Connect has an option to use typed interfaces too, so hopefully everything works as intended.